addons/mail/static/src/components/welcome_view/welcome_view.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.WelcomeView" owl="1">
+        <div class="o_WelcomeView h-100 d-flex flex-column justify-content-center align-items-center bg-light">
+            <h1 class="font-weight-light">You've been invited to a meeting!</h1>
+            <h2 class="m-5" t-esc="messaging.companyName"/>
+            <div class=" d-flex justify-content-center">
+                <MediaPreview class="mr-5" localId="props.mediaPreviewLocalId"/>
+                <div class="d-flex flex-column justify-content-center">
+                    <t t-if="messaging.currentGuest">
+                        <label class="o_WelcomeView_guestNameInputLabel" for="guest-name">What's your name?</label>
+                        <input class="form-control mb-3" type="text" placeholder="Your name" name="guest-name" id="guest-name" t-ref="guestNameInput" t-on-input="welcomeView.onInputGuestNameInput.bind(welcomeView)" t-on-keydown="welcomeView.onKeydownGuestNameInput.bind(welcomeView)"/>
```

Comment:
default value of input should be current guest.name

Reply:
also for= and id= needs to be built dynamically with a unique value, as id must be unique on a page and a component could/should be reusable

Reply:
also bind in _created()